src/mainwindow.cpp

```diff
@@ -396,17 +396,6 @@ void MainWindow::setupMainWindow()
     ui->listviewLabel2->setMinimumSize({ 40, 25 });
     ui->listviewLabel2->setMaximumSize({ 40, 25 });
 
-#ifdef __APPLE__
-    QString m_displayFont(QFont(QStringLiteral("SF Pro Text")).exactMatch()
-                                  ? QStringLiteral("SF Pro Text")
-                                  : QStringLiteral("Roboto"));
-#elif _WIN32
-    QString m_displayFont(QFont(QStringLiteral("Segoe UI")).exactMatch()
-                                  ? QStringLiteral("Segoe UI")
-                                  : QStringLiteral("Roboto"));
-#else
-    QString m_displayFont(QStringLiteral("Roboto"));
-#endif
 #ifdef __APPLE__
     QFont m_titleFont(m_displayFont, 13, QFont::DemiBold);
```

Comment:
Can we rename this to `titleFont` while we're at it? Since it's not a member variable.

Reply:
Done!